Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs

```diff
@@ -984,13 +952,12 @@ profileOptionsLayout state push =
     ] (mapWithIndex
         (\index optionItem ->
             linearLayout
-            ([ width MATCH_PARENT
+            [ width MATCH_PARENT
             , height WRAP_CONTENT
             , orientation VERTICAL
             , gravity CENTER_VERTICAL
             , onClick push $ const $ OptionClick optionItem.menuOptions
-            ] <> if (optionItem.menuOptions == DRIVER_BOOKING_OPTIONS) && (null state.data.downgradeOptions) then [alpha 0.5
```

Comment:
@ashishsingh101 is this change not required for other merchants? or is it changed for all variants

Reply:
can be use for all merchants but since requirement is only for `YATRISATHI` so adding config fir this

Reply:
not required for others, adding key configuration for this